src/pages/blog/BlogPage.tsx

```diff
@@ -40,7 +40,8 @@ export default function BlogPage() {
             >
```

Comment:
Can't we just render `Masonry` component once and conditionally render skeleton here?

Reply:
Fixed